torchgeo/datamodules/utils.py

```diff
@@ -19,6 +22,81 @@ class MisconfigurationException(Exception):
     """Exception used to inform users of misuse with Lightning."""
 
 
+class AugPipe(Module):
```

Comment:
Can't remember if I already suggested this, but alternative idea: what if we use `VHR10(transforms=...)`? Would that be the same as what `AugPipe` is designed to do? So `transforms` would be for sample-wise changes and `self.aug` would be for batch-wise changes.